libs/foundry-dev-tools/src/foundry_dev_tools/resources/__init__.py

```diff
@@ -1,5 +1,8 @@
 from foundry_dev_tools.resources.dataset import Dataset
 from foundry_dev_tools.resources.folder import Folder
+from foundry_dev_tools.resources.group import Group
+from foundry_dev_tools.resources.principal import Principal
 from foundry_dev_tools.resources.resource import Resource
+from foundry_dev_tools.resources.user import User
 
-__all__ = ["Resource", "Dataset", "Folder"]
+__all__ = ["Resource", "Dataset", "Folder", "Group", "Principal", "User"]
```

Comment:
True, I didn't think about the fact that Group/User are actually not resources until now. But it would be good to move them to another module for which I don not know a name yet. If you have suggestions let me know!